components/table/table-wrapper.js

```diff
@@ -0,0 +1,318 @@
+import '../colors/colors.js';
+import '../scroll-wrapper/scroll-wrapper.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+export const tableStyles = css`
+	.d2l-table {
+		border-spacing: 0;
+		font-size: 0.8rem;
+		font-weight: 400;
+		width: 100%;
+	}
+
+	.d2l-table > tbody,
+	.d2l-table > tfoot {
+		background-color: #ffffff;
+	}
+
+	/* all cells */
+	.d2l-table > * > tr > * {
+		border-bottom: var(--d2l-table-border);
+		font-weight: inherit;
+		height: var(--d2l-table-cell-height);
+		padding: var(--d2l-table-cell-padding);
+		text-align: left;
+		vertical-align: middle;
+	}
+	d2l-table-wrapper[dir="rtl"] .d2l-table > * > tr > * {
+		text-align: right;
+	}
+
+	/* default cells */
```

Comment:
Was able to save a lot of CSS to reverse things like borders in RTL by using the `:not([dir="rtl"])` -- very handy!